codejail/django_integration.py

```diff
@@ -5,11 +5,69 @@
 """
 
 from django.core.exceptions import MiddlewareNotUsed
-from django.conf import settings
+from django.conf import settings as django_settings
 
+import codejail.jail
 import codejail.limits
 
 
+def configure_from_settings(settings):
+    """
+
+    Configure a set of code jails and limits from a django settings file of the
+    form:
+
+        import codejail.languages
+
+        CODE_JAIL = {
+            'jails': [
+                {
```

Comment:
You might want to mention that each of these dicts is a `**kwargs` for CodeJail.configure()